src/Components/HighChartsWrapper/HighChartsWrapper.types.ts

```diff
@@ -1,15 +1,15 @@
 import { IStyle, IStyleFunctionOrObject, ITheme } from '@fluentui/react';
-import { OptionsLayoutValue } from 'highcharts';
-import { TimeSeriesData } from '../../Models/Constants';
+import { AlignValue, CSSObject, OptionsLayoutValue } from 'highcharts';
+import { TimeSeriesData } from '../../Models/Constants/Types';
 import { IDataHistoryAggregationType } from '../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
 
 export const MAX_NUMBER_OF_SERIES_IN_HIGH_CHARTS = 10;
 
 export interface IHighChartSeriesData {
-    name: string;
+    name: string; // name of the series which is used in legend and tooltip
```

Comment:
just a reminder that these comments don't show up in intellisense. The /** notation is ideal.